tests/unit/acquisition/function/test_active_learning.py

```diff
@@ -383,3 +396,121 @@ def test_integrated_variance_reduction_builder_updates_without_retracing() -> No
 
     npt.assert_array_almost_equal(acq_fn(query_at), expected)
     assert acq_fn.__call__._get_tracing_count() == 1  # type: ignore
+
+
+@pytest.mark.parametrize(
+    "at",
+    [
+        (tf.constant([[[-1.0]]])),
+        (tf.constant([[-0.5]])),
+        (tf.constant([[0.0]])),
+        (tf.constant([[0.5]])),
+        (tf.constant([[1.0]])),
+    ],
+)
+def test_bayesian_active_learning_by_disagreement(at: tf.Tensor) -> None:
+
+    search_space = Box([-1], [1])
+    x = to_default_float(tf.constant(np.linspace(-1, 1, 8).reshape(-1, 1)))
+    y = to_default_float(tf.reshape(binary_line(x), [-1, 1]))
+    model = VariationalGaussianProcess(
+        build_vgp_classifier(Dataset(x, y), search_space, noise_free=True)
+    )
+
+    p, _ = model.model.predict_y(to_default_float(at))
```

Comment:
`VariationalGaussianProcess` has `predict_y` method, this should work as well
```suggestion
    p, _ = model.predict_y(to_default_float(at))
```